client/analytics/report/revenue/index.js

```diff
@@ -300,7 +311,7 @@ class RevenueReport extends Component {
 					title={ __( 'Revenue Last Week', 'wc-admin' ) }
 					rows={ rows }
 					headers={ headers }
-					onClickDownload={ noop }
+					onClickDownload={ this.onDownload( headers, rows, totals, query ) }
```

Comment:
More a question for @LevinMedia - should we hide the download csv button on small viewports? I recall some prior convos with you around the downloading experience being different on mobile.

Reply:
@timmyc @Aljullu in the mobile experience the download button is replaced with an email button:

<img width="647" alt="screen shot 2018-08-30 at 11 27 54 am" src="https://user-images.githubusercontent.com/4500952/44871508-d0c49600-ac47-11e8-8080-82bbacc87c0b.png">

Per the discussion in p90Yrv-LB-p2 it sounds like there would be some additional work required to provide a secure link to download the CSV via email.

The updated designs jay provided in the comments on p90Yrv-LB-p2 also allow for an email to be sent via desktop as well. 

Would it be possible to update the the button style in this PR to match the designs in p6riRB-3lZ-p2

Reply:
Sounds good, but I would leave this PR as it is now (it doesn't include any 'visible' changes) and handle the button switch in a follow-up PR. How does that sound @timmyc @LevinMedia?